src/callout.tsx

```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
                 {icon && <div className={classes.icon}>{icon}</div>}
                 <div className={classes.content}>
                     <Stack space={16}>
-                        <>
-                            <Text3 as="h2" regular>
-                                {title}
-                            </Text3>
-                            <Text2 as="p" regular color={colors.textSecondary}>
-                                {description}
-                            </Text2>
-                        </>
+                        <Inline fullWidth alignItems="flex-start" space="between">
+                            <Stack space={0}>
+                                <Text3 as="h2" regular>
+                                    {title}
+                                </Text3>
+                                <div style={{height: '4px'}}></div>
+                                <Text2 as="p" regular color={colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </Stack>
+                            {onClose && (
+                                <IconButton
+                                    size={40}
+                                    style={{
+                                        margin: '-8px -12px -8px 0',
```

Comment:
better use `position: relative, top: xxx, left: xxx` to position the element

Reply:
I think we're missing something, making this change breaks the icon alignment. Not sure why, but with the margin is fine but with this change is not.